multiqc/utils/search_patterns.yaml

```diff
@@ -97,6 +97,8 @@ busco:
     fn: 'short_summary_*'
 custom_content:
     fn_re: '.+_mqc\.(yaml|yml|json|txt|csv|tsv|log|out)'
+clipandmerge:
+    contents: 'ClipAndMerge ('
```

Comment:
Is this always near the top of the file? Good to add `num_lines` if so to prevent searching huge files.

Reply:
It is always at the top - to be sure I added a `num_lines=5` .... shouldn't harm...